src/Common/MainController.h

```diff
@@ -315,7 +315,7 @@ public slots:
 
     QScopedPointer<Geo::IpToLocationResolver> ipToLocationResolver;
 
-    QList<Recorder::JamRecorder> jamRecorders;
+    QList<Recorder::JamRecorder *> jamRecorders;
```

Comment:
So should I create the (empty) list in the dialog constructor and then run through it deleting the referenced objects in the destructor?